src/_components/index.js

```diff
@@ -1 +1,3 @@
-export * from './PrivateRoute';
\ No newline at end of file
+export * from './PrivateRoute';
+export * from './headerComponent';
+export * from './vSlider';
```

Comment:
Take out this line

Reply:
on my local code, only saw exports for the headerComponents and the Private route.